exporter/collector/integrationtest/testcases/testcases_metrics.go

```diff
@@ -317,6 +318,17 @@ var MetricsTestCases = []TestCase{
 		// SDK exporter does not support CreateServiceTimeSeries
 		SkipForSDK: true,
 	},
+	{
+		Name:                 "Custom User Agent",
+		OTLPInputFixturePath: "testdata/fixtures/metrics/counter.json",
+		ExpectFixturePath:    "testdata/fixtures/metrics/counter_user_agent_expect.json",
+		ConfigureCollector: func(cfg *collector.Config) {
+			cfg.UserAgent = "custom-user-agent"
```

Comment:
Should we test the `{{version}}` interpolation here as well?

Reply:
I initially had that, but it doesn't work with the SDK exporter.  I decided to remove it from this test, as it is already demonstrated in other tests.